content/src/content/jcr_root/apps/core/wcm/components/teaser/v2/teaser/clientlibs/editor/css/teaser.less

```diff
@@ -33,3 +33,8 @@
     width: 20%;
     margin-left: @cmp-teaser__editor-actions-spacing;
 }
+
+/* stylelint-disable-next-line selector-max-type */
+.cmp-teaser__editor-action coral-tooltip {
+    width: 100px;
```

Comment:
Wouldn't `min-width` be a better property so that we don't override other styles possibly coming from Coral UI?

Reply:
Addressed in fce0c50342b85fccfcefe0c81d99a8dbe5bc2eec.